golem_messages/validators.py

```diff
@@ -19,3 +19,133 @@ def validate_integer(field_name, value):
             field=field_name,
             value=value,
         )
+
+
+def is_number_positive(value, value_type):
+    return isinstance(value, value_type) and value > 0
+
+
+def validate_positive_integer_value(field_name, value):
+    if not is_number_positive(value, int):
+        raise exceptions.FieldError(
+            "should be a positive integer field",
+            field=field_name,
+            value=value,
+        )
+
+
+def validate_not_negative_integer_value(field_name, value):
+    if not isinstance(value, int) or value < 0:
+        raise exceptions.FieldError(
+            "should be a not negative integer field",
+            field=field_name,
+            value=value,
+        )
+
+
+def validate_boolean(field_name, value):
+    if not isinstance(value, bool):
+        raise exceptions.FieldError(
+            f"{field_name} should be boolean",
+            field=field_name,
+            value=value,
+        )
+
+
+def validate_list(field_name, value, min_length, max_length):
+    if not (isinstance(value, list) and
+            (min_length <= len(value) <= max_length)):
+        raise exceptions.FieldError(
+            f"{field_name} should be a list with minimum {min_length}"
+            f" element and maximum {max_length} elements.",
+            field=field_name,
+            value=value,
+        )
+
+
+def validate_list_of_positive_integers(
+        field_name,
```

Comment:
Please fix indentation.

Reply:
Honestly I have did it in right way first, but when I run linter it gave me:
```
-      {
-          "type": "convention",
-          "module": "golem_messages.validators",
-          "obj": "",
-          "line": 68,
-          "column": 0,
-          "path": "golem_messages/validators.py",
-          "symbol": "bad-continuation",
-          "message": "Wrong hanging indentation before block (add 4 spaces).\n    field_name,\n    ^   |",
-          "message-id": "C0330"
-      },
```
That's why I left it with 8 spaces.